examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
I see.

Nevertheless, relative URLs on `localhost` or over 'https' are exempted from this check. So this PR works.

Regarding to your suggestion about having 2 examples, one with https://localhost:8000 and one with http://localhost:8000, it actually doesn't really demonstrate the requirements either. The requirement is basically saying: 1) use https. 2) for localhost, do whatever. This PR demostrate 2), but 1) is really hard to demonstrate. Because we need a non-localhost https server running.

Not sure if it is worth to start such a server on heroku?